src/genesis.js

```diff
@@ -17,10 +17,10 @@ const w3Util = (new Web3()).utils;
 /// Chain Specification Template
 const ChainSpecTemplate = require('../template.json');
 
-const ElapsedTime = 900000;
+const ElapsedTime = w3Util.toBN(863338);
```

Comment:
Where does this number come from? It would be good to add a comment as in `VestingLength`.

Reply:
It was the most recent CC2 block when I checked it before making the commit :)